src/style.css

```diff
@@ -16,24 +19,27 @@
 }
 
 body {
-  background-color: var(--background-color);
-  padding: 5px;
+  background-image: linear-gradient(to right, #4facfe 0%, #00f2fe 100%);
 }
 
 .header {
   display: var(--display-flex);
   width: 100vw;
   padding: 20px;
+  margin-bottom: 100px;
 }
```

Comment:
- [ ] There is a little overflow that we can fix easily by just changing the width of the header to a percentage instead of the view width(VW).

We want to avoid the next:
![Screen Shot 2022-01-27 at 10 59 21](https://user-images.githubusercontent.com/67211919/151408154-4f302d53-6afb-4d01-914d-f4747e0adaa2.png)

See that blue background right at the end?